pkg/controllers/update.go

```diff
@@ -69,14 +69,19 @@ func RunInstallScript() (bool, string, Error) {
 		exitCode := 1
 		if exitError, ok := err.(*exec.ExitError); ok {
 			exitCode = exitError.ExitCode()
-		}
+		}		
 
 		message := "Unable to install the latest Doppler CLI"
 		permissionError := exitCode == 2 || strings.Contains(strOut, "dpkg: error: requested operation requires superuser privilege")
 		if permissionError {
 			message = "Error: update failed due to improper permissions\nPlease re-run with `sudo` or as an admin"
 		}
 
+		gpgError := strings.Contains(strOut, "Unable to find gpg")
```

Comment:
You'll want to use a unique exit code for this scenario, like the permission error above

Reply:
Ok. I'll add a note in our documentation about this instead.